operators/src/engine/operator.rs

```diff
@@ -41,11 +43,14 @@ pub trait RasterOperator:
     /// execution context. Instead, `_initialize` should be implemented.
     async fn initialize(
         self: Box<Self>,
+        path: WorkflowOperatorPath,
         context: &dyn ExecutionContext,
     ) -> Result<Box<dyn InitializedRasterOperator>> {
         let span = self.span();
-        let op = self._initialize(context).await?;
-        Ok(context.wrap_initialized_raster_operator(op, span))
+        debug!("Initialize {:?}, path: {:?}", self.typetag_name(), &path);
+        let op = self._initialize(path.clone(), context).await?;
```

Comment:
I ran the Python WMS example and it output

> geoengine_operators::engine::operator: Initialize "GdalSource", path: WorkflowOperatorPath { id: [] }

why is the path empty?

Also, here and in the Wrapper adapter, I guess we should not do debug output, but format it by joining the path components with ","

Reply:
yes that is correct. The root is "[]".
i added display here: https://github.com/geo-engine/geoengine/pull/780/commits/b286ed0c8e8a84e6f3111f7f9605a1184529f5fd